src/main/java/bio/overture/ego/controller/ApiKeyController.java

```diff
@@ -229,10 +234,10 @@ public ApiKeyController(
 
   /** DEPRECATED: GET /token to be removed in next major release */
   @Deprecated
+  @AdminScoped
```

Comment:
Is this a breaking change? Given that the first line of the method is a method call to `checkAdminOrOwner` emphasis on the word *owner*

Reply:
Yea if only ADMINs can list tokens, then this breaks the functionality in ARGO of users listing their own.

Reply:
GOOD EYE